src/converters/building_converter.py

```
@@ -1,24 +1,99 @@
+            # 4. 移除首尾空格（作为保险）
+            key = key.strip()
+            return key
+
+        # 使用字典推导式，遍历原始数据，生成一个键名干净的新字典
+        # 例如： "North Axis {deg}" -> "North_Axis"
+        cleaned_building_data = {clean_key(key): value for key, value in building_data.items()}
+        
+        self.logger.info(f"      - Cleaned data keys: {list(cleaned_building_data.keys())}")
+
+
+        # --- 现在，使用这个“干净”的字典来创建IDF对象 ---
```

Ruff (0.13.1)

Comment contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF003)